etc/docker/dev/docker-compose.yml

```diff
@@ -79,3 +79,15 @@ services:
     volumes:
       - ../../certs/hostcert_xrd3.pem:/tmp/xrdcert.pem
       - ../../certs/hostcert_xrd3.key.pem:/tmp/xrdkey.pem
+  minio:
```

Comment:
you need to add this to the links: up above in the file.

Reply:
Thanks, not sure how I missed that. Just fixed it.